srcpkgs/pymol/template

```diff
@@ -0,0 +1,24 @@
+# Template for 'pymol'
+pkgname=pymol
+version=2.1.0
+revision=1
+wrksrc=${pkgname}
+build_style="python3-module"
+hostmakedepends="msgpack-devel python3-numpy freetype-devel libfreeglut-devel glew-devel python3-devel libxml2-devel python3-PyQt5"
+depends="freetype glew libfreeglut python3-numpy tcsh python3-pmw python3-tkinter python3-PyQt5"
+short_desc="PyMOL molecular visualization system"
+homepage="https://pymol.org/"
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+license="pymol"
+only_for_archs="i686 x86_64"
```

Comment:
no

Reply:
Why not? The musl conditional was failing due to the indentation issue, so this seemed like a simpler solution.

Reply:
OK I've replaced with the musl conditional in the hope this time the tabs done in Leafpad work.

Reply:
pymol is available on more arches, if musl is broken beyond trivial patching then it is better to

```
case "$XBPS_TARGET_MACHINE" in
    *-musl) broken="requires extensive patching" ;;
esac
```